src/libraries/System.Memory/tests/ArrayBufferWriter/ArrayBufferWriterTests.T.cs

```diff
@@ -205,6 +205,17 @@ public void GetMemory_DefaultCtor(int sizeHint)
             Assert.Equal(sizeHint <= 256 ? 256 : sizeHint, memory.Length);
         }
 
+        [ConditionalFact(nameof(IsX64))]
```

Comment:
Since we are trying to avoid the OOM, don't run this on Linux.
https://github.com/dotnet/runtime/blob/d59f143221bfa500fe2b974827dcb47eb4ce327c/src/libraries/System.Text.Json/tests/Serialization/WriteValueTests.cs#L384-L388